### ISSUE TITLE
Stylesheets: Displaying Metadata Information

### ISSUE THREAD

**Issue by taroth21** [openSUSE/suse-xsl#35]
The new stylesheets should be able to display the following metadata:
- file name
- file maintainer
- doc status
- ID  
- "last changed" date for the file
- URL to the document (SVN or GitHub), see --repository

Maybe it would also be useful to display all metadata that is available within the document (for example, as a table: key - value)

**Comment by tomschr** [comment]
Ok, these properties can be retrieved easily:

* file name: can be retrieved from `xml:base` attribute (is automatically inserted during profiling)
* file maintainer and doc status from DocManager's elements
* ID from `xml:id`

This needs a change in our SUSEDoc schema:
* URL to the document: I would use one of bibilographic elements like `biblioid`, `bibliosource` etc.

This is impossible to get through usual XSLT transformation:

* "last changed" date of a file

**Comment by tomschr** [comment]
This issue relates to openSUSE/daps#281

**Comment by tomschr** [comment]
The file maintainer, doc status, and ID can probably be retrieved from DocManager tags. For "last changed" date for the file and file name, I assume, you need to enhance DAPS.

**Comment by sknorr** [comment]
Tom, what is with the commits from July 16 shown here that implement VCS URLs via bibliosource? I think your last commit means that you now have implemented this feature twice, unfortunately.
(d43c32a, f1c33e5)

Also, the "last changed" date thing is not fixed.

=> Reopening.

**Comment by tomschr** [comment]
@sknorr Yes, that was a mistake. I used "fix" instead of "issue". :cry: 

Apart from that, I'm wondering if I should remove the `bibliosource` element and better rely on those DocManager elements.

**Comment by sknorr** [comment]
I'd say yes, go for it...

**Comment by tomschr** [comment]
Some issues are not fixed, so reopen (accidently closed)

**Comment by tomschr** [comment]
@taroth21 Is this still an issue? The current 2022 HTML stylesheet doesn't show any information at all.

**Comment by tomschr** [comment]
After this long time, we can close it. Reopen it if it's still an issue.
